src/clr-angular/modal/utils/ghost-page-animations.ts

```diff
@@ -3,7 +3,11 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-export const GHOST_PAGE_ANIMATION = {
+
+/*
+ * @deprecated since 0.12
+ */
+export const GHOST_PAGE_ANIMATION_DEPRECATED = {
```

Comment:
No reason to change this name, its internal and should just have a deprecated comment.